README.md

```diff
@@ -1,5 +1,13 @@
 # Encoda
 
+## Announcement
+
+For some time our main focus has been a rewrite of [our Stencila platform](https://github.com/stencila/stencila) (`v2`). We are porting over codecs and other functionality from this repo to the `v2` of our Stencila platform.
+
+This means that we are unable to be so active with contributions to this project. eLife have volunteered to continue development on this repository until such a time as the `v2` of Stencila can meet their needs. They only have need of the codecs to convert from JATS to JSON.
+
+From `v2.0.0` of this project we will only continue to support the conversion from JATS to JSON. If you want to use any other formats then please see if they are available in Stencila or use the [v1.0.3](https://github.com/stencila/encoda/releases/tag/v1.0.3) release.
```

Comment:
@nokome I drafted this announcement at the top of the README. Rather than give feedback just change it to something you are happy with. I just wanted to start you off with something.

If you are also able to please just remove any sections of the README that are no longer relevant. For our needs we may just put in an example of how we are using it and remove everything else.

Reply:
I've decided to word it in a way that permits us not to focus on a README rewrite at this time.